src/main/kotlin/no/skatteetaten/aurora/boober/feature/BigIpLegacyFeature.kt

```diff
@@ -0,0 +1,90 @@
+package no.skatteetaten.aurora.boober.feature
+
+import com.fkorotkov.kubernetes.newObjectMeta
+import no.skatteetaten.aurora.boober.model.AuroraConfigFieldHandler
+import no.skatteetaten.aurora.boober.model.AuroraContextCommand
+import no.skatteetaten.aurora.boober.model.AuroraDeploymentSpec
+import no.skatteetaten.aurora.boober.model.AuroraResource
+import no.skatteetaten.aurora.boober.model.openshift.BigIp
+import no.skatteetaten.aurora.boober.model.openshift.BigIpKonfigurasjonstjenesten
+import no.skatteetaten.aurora.boober.model.openshift.BigIpSpec
+import no.skatteetaten.aurora.boober.utils.addIfNotNull
+import org.apache.commons.codec.digest.DigestUtils
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+
+@Service
+class BigIpLegacyFeature(
```

Comment:
Utifra hva jeg kan se så virker denne litt redundant siden den bruker BigIpFeature i stor grad bortsett fra i generate, men BigIpFeature har lagt opp til at det er mulig med legacy i generate. 
Hvis du vil beholde legacyfeature så ville jeg vurdert å flytte mer over hit, f.eks handlers og validering av service for legacy. I tilleg ville jeg fjernet noe av den koden i bigip feature som omhandler legacy.

Reply:
Det jeg egentlig er ute etter er at skillet mellom disse to blir litt klarere og adskilt, evt kombinere de til en feature.

Reply:
Jeg hadde dem som en feature i starten, men generate metoden ble altfor komplisert til å skulle støtte begge delere. Det vil gå fint å flytte legacy-handlerne fra BigIPFeature til BigIPLegacyFeature, men er ikke gunstig å splitte validate slik jeg ser det. Så da valgte jeg å beholde legacy handlerne i BigIPFeature for lettere oversikt over handlere som blir brukt i validate.